src/app/components/trip-details/trip-details.component.html

```diff
@@ -6,6 +6,45 @@
   @if (trip$ | async; as trip) {
     <div class="flex flex-col gap-2 p-4">
       <h1 class="text-2xl font-bold">{{ trip.title }}</h1>
+      <nz-calendar [nzFullscreen]="false" [nzValue]="trip.startDate">
+        <ul *nzDateFullCell="let date" class="max-h-14 overflow-hidden">
+          @if (date && trip.startDate && trip.endDate) {
+            @if (
+              getDateWithoutTime(date).valueOf() >=
+                getDateWithoutTime(trip.startDate).valueOf() &&
+              getDateWithoutTime(date).valueOf() <=
+                getDateWithoutTime(trip.endDate).valueOf()
+            ) {
+              <li
+                class="mb-1 ml-auto mr-auto h-6 w-6 rounded-full bg-warning px-1">
```

Comment:
I think you can join `ml-auto mr-auto` to `mx-auto`

Reply:
I thought so as well, tried it and it didn't work 🥲

Reply:
Its like that sometimes </3